src/encoded/static/components/genome_browser.js

```diff
@@ -49,17 +50,19 @@ const readGenomeBrowserLabelCoordinates = () => {
  * @param {boolean} [ignoreCache=false] True to not look into cache, false to use cache
  * @returns Default coordinates
  */
-const getDefaultCoordinates = (assemblyAnnotation, ignoreCache = false) => {
+const getDefaultCoordinates = (assemblyAnnotation, geneAnnotation, ignoreCache = false) => {
     const assembly = assemblyAnnotation.split(' ')[0];
+    const annotation = geneAnnotation.split(' ')[0];
     // Files to be displayed on all genome browser results
```

Comment:
Do you need to split these? I think these are already split now?

Reply:
You are right. No need. I didn't fully understand what it is.